.github/unittest.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo '::group::Prepare conda'
```

Comment:
This line as well as the accompanying `echo '::endgroup::'` is used to group the output of some steps in the logs. This is only needed now, since we no longer have individual steps, but rather a single step.